### PR TITLE
[NCP-3449] added a sentence for waiting verification

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: zh-Hans.json [hunk 1]
```diff
@@ -261,6 +261,7 @@
 	"certificate.dns01challenge.steps.s4.title": "证书上传",
 	"certificate.dns01challenge.steps.try.again.message": "抱歉，DNS挑战失败，请稍后再试。",
 	"certificate.dns01challenge.wait.above.message": "要验证证书主机名的所有权，请添加以下 DNS TXT 记录：",
+	"certificate.dns01challenge.wait.after.start.above.message": "请稍候，我们正在验证您的 DNS 更新并创建您的新证书。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:264]: Please change the text to:
"请稍候，我们正在验证DNS，验证通过后将创建新证书。"

**Comment by Jianghai-Quantil** [comment]
@jungho73 As commented in pull request 396, please change the text in Chinese for the key: "edgeHostname.history.inprogress.not.clone.message"

AS-IS: "边缘主机名的部署正在进行中。请等待部署完成后再进行进一步更新。"
TO-BE: "该调度域名有一个部署任务正在进行中。请等待部署完成后再进行更新操作。"

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Some changes requested

File: zh-Hans.json (hunk 1 above)
Comment (jungho73) [reply zh-Hans.json]: changed it you suggested

**Comment by jungho73** [comment]
@Jianghai-Quantil 
Two sentences you suggested changed.
Thanks.

**Review by Jianghai-Quantil — APPROVED** [review]
The changes look good to me

**Pull request merged**